docs/application/dotnet/guides/user-interface/nui/text.md

```diff
@@ -312,10 +312,169 @@ The following markup elements are currently supported:
+
+    The following example shows how to apply it:
+
+    ```xaml
+    <base:TextLabel x:Name="txt" EnableMarkup="True"
+        Text="<u color='green' height='2.0f' type='dashed' dash-gap='2.0f' dash-width='3.0f'>Hello world</u>"/>
+    ```
+
+-   `<s>`
+
+    Sets the strikethrough values for the characters inside the element.
+
+    The following attributes are supported:
+
+    -   `color`: The color of strikethrough.
+    -   `height`: The height of strikethrough.
+
+    The following example shows how to apply it:
+
+    ```xaml
+    <base:TextLabel x:Name="txt" EnableMarkup="True"
+        Text="<s color='green' height='2.0f' >Hello world</s>"/>
+    ```
+
+-   `<char-spacing>`
+
+    Sets the character spacing values for the characters inside the element.
+
+    Use the `value` attribute to define the spaces between characters in Pixels.
+
+    A positive value will make the characters far apart (expanded) and a negative value will bring them closer (condensed).
+
+    The following example shows how to apply it:
+
+    ```xaml
+    <base:TextLabel x:Name="txt" EnableMarkup="True"
+        Text="<char-spacing value='5.0f'>Hello world</char-spacing>"/>
+    ```
+
+-   `<p>`
+
+    Use paragraph tag to set many styles on paragraph's level for the lines inside the element.
+
+    The following attributes are supported:
+
+    -   `align`: Use the `align` attribute to define the horizontal alignment of paragraph. The supported attribute values are `begin`, `center` and `end` .
+    -   `rel-line-height`: The relative height of the line (a factor that will be multiplied by text height).
+
+    The following example shows how to apply alignment:
+
+    ```xaml
+    <base:TextLabel x:Name="txt" EnableMarkup="True"
+        Text="text outside<p align='end'>Paragraph end</p>text outside<p align='center'>Paragraph center</p>text outside<p align='begin' >Paragraph begin</p>"/>
+    ```
+
+    The following example shows how to apply relative height of line:
+
+    ```xaml
+    <base:TextLabel x:Name="txt" EnableMarkup="True"
+        Text="<p>line 1</p><p rel-line-height=0.5>line\n2</p>line 3<p rel-line-height=3>line\n4</p>line 5"/>
+    ```
+
+-   `<item>`
+
+    Defines an embedded item within the text.
+
+    The following attributes are supported:
+
+    -   `url`: The url path of the image.
+    -   `width`: The width of the item.
+    -   `height`: The height of the item.
+    -   `color-blending`: Use the color-blending attribute to define whether the color of the image is multiplied by the color of the text.
+
+    The following example shows how to apply it:
+
+    ```xaml
+    <base:TextLabel x:Name="txt" EnableMarkup="True"
+        Text="<item 'width'=26 'height'=26 'url'='path/image.png' 'color-blending'='multiply'/>"/>
+    ```
+    > [!NOTE]
+    > The url of the image is optional. If there is no image the layout engine will use the width and height to create a space inside the text. This gap can be filled later.
```

Comment:
URL should be capitalized here. Please look into it.
```suggestion
    > The URL of the image is optional. If there is no image the layout engine will use the width and height to create a space inside the text. This gap can be filled later.
```

Reply:
Done